packages/react/src/primitives/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
```

Comment:
I was thinking if this is over engineering..But looks like we can just export the `Checkbox`, just follow the same patterns as how we build other fields.